### PR TITLE
Publish dist lol

### PR DESCRIPTION
**Pull request by koddsson**
Added `dev-server-storybook` to the root `tsconfig.json`.

### CODE REVIEW

**Review by bashmish — APPROVED** [review]

**Comment by koddsson** [comment]
@bashmish:

Any idea on how I can make sure `module` and `moduleResolution` are set to `node16`? Running the steps you mentioned makes the following diff:

```diff
index d23e2f60..7430b12e 100644
--- a/packages/dev-server-storybook/tsconfig.json
+++ b/packages/dev-server-storybook/tsconfig.json
@@ -3,8 +3,7 @@
 {
   "extends": "../../tsconfig.node-base.json",
   "compilerOptions": {
-    "module": "node16",
-    "moduleResolution": "node16",
+    "module": "commonjs",
     "outDir": "./dist",
     "rootDir": "./src",
     "composite": true,
```

**Comment by koddsson** [comment]
Looks like some projects like `@web/mocks` don't use the `workspace-packages` stuff so I think it's good that we don't either.

**Comment by bashmish** [comment]
@koddsson oh, I see you updated the main "tsconfig.json"
this one will update on `npm run update` next time and remove your package again, maybe that's what happened recently

**Comment by koddsson** [comment]
Hmm, but it needs to be in there so it's built during `npm run build` :(

Let me try to figure stuff out.

**Comment by bashmish** [comment]
@koddsson min Node is now 16, right? can we just update it in all tsconfigs then? I'll make another PR

File: packages/dev-server-storybook/tsconfig.json [hunk 1]
```diff
@@ -1,5 +1,3 @@
-// Don't edit this file directly. It is generated by generate-ts-configs script
-
 {
   "extends": "../../tsconfig.node-base.json",
   "compilerOptions": {
```
Comment (bashmish) [inline packages/dev-server-storybook/tsconfig.json:3]: the lines are gone now :(

Comment (koddsson) [reply packages/dev-server-storybook/tsconfig.json:3]: Yeah let me revert back to a good state.

**Review by bashmish — APPROVED** [review]

**Comment by bashmish** [comment]
> > @koddsson min Node is now 16, right? can we just update it in all tsconfigs then? I'll make another PR
> 
> I created a PR to update to Node 18 a while back. We aren't running node 16 in CI anyways.

do IIUC that we generate Node16-compatible JS while having Node18 minimum requirement?

**Comment by koddsson** [comment]
Ah I think the issue of `npm update` blowing away the reference in `tsconfig.json` is still present here though. Working on that now.

**Pull request merged**